portal-web/test/functional/com/liferay/portalweb/tests/UncheckDisableSPAInInternetExplorer.testcase

```diff
@@ -21,9 +20,10 @@ definition {
 			configurationName = "Frontend SPA Infrastructure",
 			configurationScope = "System Scope");
 
-		FrontendSPAInfrastructure.UncheckDisableSPA();
+		FrontendSPAInfrastructure.uncheckDisableSPA(
+			settingName = "Disable SPA in Internet Explorer");
 
-		FrontendSPAInfrastructure.Save();
+		PortleEntry.save();
```

Comment:
@PablitoBonito , I found a better suited macro that also uses update: see SystemSettings.saveConfiguration macro.